tron/core/jobrun.py

```diff
@@ -111,13 +111,15 @@ def to_json(state_data: dict) -> Optional[str]:
 
     @staticmethod
     def from_json(state_data: str):
-        """Deserialize the JobRun instance to a JSON string."""
+        """Deserialize the JobRun instance from a JSON string."""
         try:
             json_data = json.loads(state_data)
-            if json_data["run_time"]:
-                run_time = datetime.datetime.fromisoformat(json_data["run_time"])
+            run_time_str = json_data["run_time"]
+            if run_time_str:
+                run_time = datetime.datetime.fromisoformat(run_time_str)
```

Comment:
i'm personally not a fan of adding a suffix to variables with their type and would propose something like
```suggestion
            raw_run_time = json_data["run_time"]
            if raw_run_time:
                run_time = datetime.datetime.fromisoformat(raw_run_time)
```
but i won't block on this :)